.pa11yci--mobile

```diff
@@ -0,0 +1,13 @@
+{
+  "defaults": {
+    "userAgent": "pa11y",
+    "timeout": 120000,
+    "concurrency": 8,
+    "standard": "WCAG2AA",
+    "runners": ["axe"],
+    "viewport": {
```

Comment:
Mobile pa11y settings are the same as the default, but include a special mobile viewport